Add CONTRIBUTING docs (#43)
paralus/helm-charts: CONTRIBUTING.md

You'll need to perform the following tasks in order to submit your changes:

- Fork the paralus/helm-charts repository.
- Create a branch for your changes.
- Add commits to that branch.
- Open a PR to share your contribution.

Below are all the details you need to know and get started with the development.

# Paralus helm-charts

This repo hosts the helm charts that is used to deploy `paralus` to any kubernetes cluster.

## Development setup

### Installing and Configuring Kind

If you don't already have Kind installed on your local system, you can do so by following the [Kind Quickstart Documentation](https://kind.sigs.k8s.io/docs/user/quick-start/). The default settings are good enough to get you started.

The next step is to **create a Kind cluster**. To do that you can create a copy of [this configuration file](https://github.com/paralus/helm-charts/blob/main/docs/kind.config.yaml) and use that to create a cluster.

```bash
kind create cluster --config cluster.yaml
```

*You can optionally pass name via `--name <name>` to create a kind cluster with a specific name.*

> ### Install paralus from helm chart source

Reviewer: ```suggestion
### Install Paralus from helm chart source
```

You can install the helm chart from source using the following command once you have cloned the repo.

``` bash
# helm install -n <namesapce> <installation_name> ./charts/ztka --values <custom_values_file> --create-namespace
helm install -n ztka ztka ./charts/ztka --values examples/values.dev-generic.yaml --create-namespace
```

> You should now have `paralus` deployed. You can check out [blog](https://www.paralus.io/blog/kind-quickstart) for a detailed guide on how to set up the rest of the application.

Reviewer: ```suggestion
You should now have `paralus` deployed. You can check out our blog post to [setup Paralus on Kind](https://www.paralus.io/blog/kind-quickstart) for a detailed guide on how to set up the rest of the application.
```

From here you should be able to edit the files and repeat these steps to test it out.

## Docuemntation

The documentation for each chart is done with [helm-docs](https://github.com/norwoodj/helm-docs). This way we can ensure that values are consistent with the chart documentation.

Run the following command to regenerate the docs. Make sure to run this before you submit a pull request.

``` shell
helm-docs --chart-search-root=charts
```

## Need Help?

> If you are interested to contribute to prompt but are stuck with any of the steps, feel free to reach out to us. Please [create an issue](https://github.com/paralus/relay/issues/new) in this repository describing your issue and we'll take it up from there.

Reviewer: Please correct the issue link
```suggestion
If you are interested to contribute to prompt but are stuck with any of the steps, feel free to reach out to us. Please [create an issue](https://github.com/paralus/helm-charts/issues/new) in this repository describing your issue and we'll take it up from there.
```

> If you are interested to contribute to prompt but are stuck with any of the steps, feel free to reach out to us. Please [create an issue](https://github.com/paralus/helm-charts/issues/new) in this repository describing your issue and we'll take it up from there.

Reviewer 2: I think you meant if you are interested to contribute to helm-charts ?

Author: Updated
